### DISCUSSION TITLE
Mark a folder as read with a button?

### DISCUSSION THREAD

**Discussion by mariomaz87** [martinrotter/rssguard#1547] [Q&A] [answered]
Hi, is there a way to mark an entire folder as read, using buttons? Currently I can right click on the folder and mark as read, or select all the articles and then mark as read using the toolbar button. Thanks!

**Comment by Ac314** [comment]
Toolbar for articles list -> Mark selected items read

**Reply by martinrotter** [reply]
you can
![rssguard_tDm7EyVDfa](https://github.com/user-attachments/assets/72852231-e8d6-407f-8773-dd64ac30bab5)
you can change the shortcut in settings

**Comment by mariomaz87** [comment]
Sorry, my request was not very clear. I would like a button like these, to mark all the articles in the folder I'm in as read: 
![Screenshot 2024-12-09 114947](https://github.com/user-attachments/assets/1035ae79-bdf2-4da8-8ba1-e51d3f865536)

Right now Mark articles read button applies only to the selected articles\articles

**Reply by Ac314** [reply, marked as answer]
> Right now Mark articles read button applies only to the selected articles\articles

Not "Mark articles read" button. "Mark selected items read" is a different button.

**Reply by martinrotter** [reply]
Go to main menu Feeds -> Mark selected items read

This action marks selected item in FEED LIST as read. So if you select category, it marks full category as read. You can assign shortcut to this action in settings. You can also move this action to FEED toolbar if you wish, also in settings.

**Reply by mariomaz87** [reply]
Thanks, now I get it :D
